l10n_cn_fapiao/data/fapiao.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="fapiao_tax_type_13" model="fapiao.tax.type">
```

Comment:
Over time, some new taxes might replace old ones. It is not frequent but can happen.
In this case we cannot delete the old ones or rename them because they are used in old history record.
We need to be able to deactivate them so that people do not use it for new fapiao.

You need to add a domain control in the field and in the view (by default shows active only).

Let me know if this is clear enough @Wizr